uicomponent-compose/core/build.gradle

```diff
@@ -11,6 +11,7 @@ apply from: rootProject.file("gradle/android.gradle")
 
 android {
     compileOptions {
+        coreLibraryDesugaringEnabled true // need for test. https://github.com/DroidKaigi/conference-app-2021/issues/373
```

Comment:
Required for testing. Described in the following document.

https://developer.android.com/studio/write/java8-support#library-desugaring

```
Note that you may also need to include the above code snippet in a library module's build.gradle file if:

The library module's instrumented tests use these language APIs (either directly or through the library module or its dependencies). This is so that the missing APIs are provided for your instrumented test APK.
```